Sources/Shared/Data Manager/Model/Social/SocialComment.m

```diff
@@ -79,18 +99,88 @@ -(void) parseTextHTML {
             if (!self.imageURLs){
                 self.imageURLs = [[NSMutableArray alloc] init];
             }
-            [self.imageURLs addObject:imgTag];
+            NSString * absoluteImgSrc = [self absolutePathFromStringURL:imgTag];
+            self.text = [self.text stringByReplacingOccurrencesOfString:imgTag withString:absoluteImgSrc];
+            NSURL * urlTest = [NSURL URLWithString:absoluteImgSrc];
+            if (urlTest){
+                [self.imageURLs addObject:absoluteImgSrc];
+            }
         }
         range = [htmlString rangeOfString:@"<img[^>]+src=\"" options:NSRegularExpressionSearch];
     }
- 
-    self.message = [self.text stringByConvertingHTMLToPlainText];
+    
+    
+    self.message = [self plainTextFormHTML:self.text];
     self.cellHeight = 0;
 }
 
--(NSString *) toHTML {
+-(NSAttributedString *) plainTextFormHTML:(NSString *) htmlString {
+    if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"8.3")){
+        
+        NSString * string = htmlString;
+        NSData *HTMLData = [string dataUsingEncoding:NSUTF8StringEncoding];
+        NSMutableAttributedString *attrString = [[NSMutableAttributedString alloc] initWithData:HTMLData options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType} documentAttributes:NULL error:NULL];
+
+        [attrString beginEditing];
+        [attrString enumerateAttribute:NSFontAttributeName inRange:NSMakeRange(0, attrString.length) options:0 usingBlock:^(id value, NSRange range, BOOL *stop) {
+            if (value) {
+                UIFont *oldFont = (UIFont *)value;
+                /*----- Remove old font attribute -----*/
+                [attrString removeAttribute:NSFontAttributeName range:range];
+                //replace your font with new.
+                /*----- Add new font attribute -----*/
+                if ([oldFont.fontName isEqualToString:@"TimesNewRomanPSMT"])
+                    [attrString addAttribute:NSFontAttributeName value:[UIFont systemFontOfSize:14] range:range];
```

Comment:
This is done in 2 different `if` and in `else` so it can be merged in the `else` block to reduce the code.